src/ffi/cxx.h

```diff
@@ -55,8 +55,8 @@ namespace rust {
             static String lossy(const char16_t *) noexcept;
             static String lossy(const char16_t *, std::size_t) noexcept;
 
-            String &operator=(const String &) &noexcept;
-            String &operator=(String &&) &noexcept;
+            String &operator=(const String &) & noexcept;
```

Comment:
hmm.. what happend on the format? we do 
```
PointerAlignment: Right
ReferenceAlignment: Pointer
```
https://github.com/DeFiCh/ain/blob/ce178e6711de32390ed3d166e4b1d7012bc853b2/src/.clang-format#L138

Reply:
For some reason, when I was trying to push my changes, I was getting `clang format` error on some of the cxx.h in ffi, so I used `./masksh format` In my VS_Code, it removed unwanted space.